docs/BackupAndRestore.md

```diff
@@ -594,18 +630,20 @@ drwxr-xr-x  2 nuodb root 4096 Feb 19 10:32 demo-save-20210219T103236
 
```

Comment:
Add the troubleshooting section into the table of contents

Reply:
We actually don't have a ToC on the page. The _Introduction_ section talks about different B&R mechanisms and includes links to the headers. Should I add a ToC at the top of the page? The guide became almost 1000 lines so it might be useful.

Reply:
And it will only grow with more examples. With a ToC you can add  sub sections so a user can go directly to a particular part of the guide based on the name of the section heading without having to scroll until they find a sub section that they need?

Reply:
I'll add a ToC in the beginning.